src/platform/site-wide/sass/style-consolidated.scss

```diff
@@ -42,7 +42,7 @@ $font-stack: "Source Sans Pro", "Helvetica Neue", "Helvetica", "Roboto", "Arial"
   font-family: "Source Sans Pro";
   font-style: normal;
   font-weight: 300;
-  src: url(/generated/sourcesanspro-light-webfont.eot?#iefix) format("embedded-opentype"),
+  src: url("/generated/sourcesanspro-light-webfont.eot?#iefix") format("embedded-opentype"),
```

Comment:
Is this needed? Looks fine, but was it causing a problem before?

Reply:
Hey @cvalarida! For some reason, the linter was screaming at me about this.

Reply:
Reverted!